### ISSUE TITLE
When we restart a batch calculation, we should use the new max parallalel calculation

### ISSUE THREAD

**Issue by greenscientist** [chairemobilite/transition#1083]
After an instance restart, if we changed the config to enable more max parallel calculation, a saved batch calculation will still use the previsouly saved max parallel. 

If the user did not specify a max value, we should always use the default one, and not save it and reuse it later.

**Comment by tahini** [comment]
For this issue, we just need to make sure that in the attributes saved for the jobs, the max parallel calculations is kept as `undefined` if not explicitly set in the UI, so that it takes the server's default value when the job executes.

The second approach would be to eliminate the max parallel calculation attribute completely. When we had to start a number of instances of trRouting, it might have been a good idea to set this value, but with the multi-thread trRouting, do we still it? @kaligrafy, thoughts?

**Comment by greenscientist** [comment]
Yes, we still need it to not destroy your CPU! We need it to throttle how many calculation we process and send in parallel, otherwise we will send too many to trRouting and cause socket starvation. 
It's also a good way to share the CPU ressource between users

**Comment by tahini** [comment]
I don't mean the maxParalllelCalculation server config, that we do need to keep. But what about the per calculation attribute, is this one necessary?

**Comment by greenscientist** [comment]
ah ok. 
It could have been usefull for people running it locally and wanted to run something in the background at a lower priority. 
But we could probably remove it for now and add something later if some users wants it. It could be in a different form then, once we rework how we manage the work queue.
